public/app/features/admin/migrate-to-cloud/EmptyState/InfoPane1.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Box, Stack } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+
+import { InfoItem } from './InfoItem';
+
+export const InfoPane1 = () => {
```

Comment:
Wonder if this could have more descriptive name, preferably without using numbers? Maybe at least Left/RightPane to show their positions?

Reply:
haha i was trying to avoid left/right because they wrap on smaller screens 😄 but i think you're right actually, 90% of the time it's gonna be left/right